### PR TITLE
MQE-1763: Investigate and fix failure due to absence of AcceptanceTester class

### PR DESCRIPTION
**Pull request by tomreece**
### Description
This bug was reproducible on branches that had updated the `symfony/yaml` package in composer from v3.4.30 to v4.3.4. The updated package was less forgiving about our functional.suite.yml file which did not properly quote string values in some cases.

This PR only exists to ensure that in the future we get better output from the `mftf build:project` command. This output was hidden in the Jenkins job because the `--verbose` flag was not used. If we had that output in Jenkins, then this bug would have been trivial to identify.

I also added troubleshooting steps to devdocs for users on versions lower than MFTF 2.5.0.

TLDR: No bug here. I increased output from `mftf build:project` and wrote some docs.

### Fixed Issues (if relevant)
1. https://jira.corp.magento.com/browse/MQE-1763

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -2,60 +2,64 @@
 
 Having a little trouble with the MFTF? See some common errors and fixes below.
 
-## WebDriver issues
```
Comment (tomreece) [inline docs/troubleshooting.md]: This documentation change looks more complicated than it is.

I discussed with @dobooth and we agreed to remove the two existing troubleshooting items on this page because they were no longer relevant.
